stdlib/shutil.pyi

```diff
@@ -103,11 +103,11 @@ def register_archive_format(
 def unregister_archive_format(name: str) -> None: ...
 
 if sys.version_info >= (3, 7):
-    def unpack_archive(filename: StrPath, extract_dir: StrPath | None = ..., format: str | None = ...) -> None: ...
+    def unpack_archive(filename: StrOrBytesPath, extract_dir: StrPath | None = ..., format: str | None = ...) -> None: ...
 
 else:
     # See http://bugs.python.org/issue30218
-    def unpack_archive(filename: str, extract_dir: StrPath | None = ..., format: str | None = ...) -> None: ...
+    def unpack_archive(filename: str | bytes, extract_dir: StrPath | None = ..., format: str | None = ...) -> None: ...
```

Comment:
Since python3.6 is end of life now and this else clause only affects python<=3.6 maybe we want to remove it? Not sure what your policy on this is so just checking 😄

Reply:
Typeshed is supporting 3.6 for a while longer to give people more time to switch 🙂 https://github.com/python/typeshed/issues/6189

Reply:
Perfect thanks! 😄